sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreCheckpointFormatV2Suite.scala

```diff
@@ -622,8 +622,8 @@ class RocksDBStateStoreCheckpointFormatV2Suite extends StreamTest
     val metadata = commitLog.get(Some(0), Some(1)).map(_._2)
 
     val versionToUniqueIdFromCommitLog = metadata.zipWithIndex.map { case (metadata, idx) =>
-      // Use stateUniqueIds(0) because there is only one state operator
-      val res2 = metadata.stateUniqueIds(0).map { uniqueIds =>
+      // Use stateUniqueIds.head(0) because there is only one state operator
+      val res2 = metadata.stateUniqueIds.head(0).map { uniqueIds =>
```

Comment:
I know Option.head would work, but probably better to explicitly use `.get` to clarify the meaning.

Reply:
Never mind it was merged already.

Reply:
Oh, sorry, if you have a follow-up opinion, feel free to make a PR, @HeartSaVioR .

Reply:
Just as I was about to reply, I noticed it had already been merged. 😂

Reply:
It's a nit, let's leave it as it is and revisit someone finds it uneasy to reason about.

Reply:
> It's a nit, let's leave it as it is and revisit someone finds it uneasy to reason about.

OK